problems/1.types+operators.js

```diff
@@ -3,15 +3,19 @@
 * @returns {boolean} - Is val a number
 */
 function isNumber() {
-
+let val = 1
+  return val
```

Comment:
This function will only ever return 1. We need to receive a number as an argument and deterimine if that argument is in fact a number. Use typeof and check that it's equal to "number'